DSCResources/MSFT_xDSCWebService/SChannel.psm1

```diff
@@ -0,0 +1,251 @@
+# This module file contains a utility to perform SChannel setup
+# Module exports XXXXXX function to perform the SChannel setup and test
+#
+# Copyright (c) Microsoft Corporation, 2016
+#
+
+# ============ Up to Date Security Settings Block =========
+$insecureProtocols            = @("SSL 2.0", "SSL 3.0", "TLS 1.0", "PCT 1.0", "Multi-Protocol Unified Hello")
+$secureProtocols              = @("TLS 1.1", "TLS 1.2")
+
+$insecureCiphers              = @('DES 56/56', 'NULL', 'RC2 128/128', 'RC2 40/128', 'RC2 56/128', 'RC4 128/128', 'RC4 40/128', 'RC4 56/128', 'RC4 64/128', 'RC4 128/128')
+$secureCiphers                = @('AES 128/128','AES 256/256','Triple DES 168/168')
+
+$enableHashes                 = @("SHA","SHA256","SHA384","SHA512")
```

Comment:
SHA1 isn't considered state of the art anymore. I'd be good to remove it from the list.